pkg/plugin/server.go

```diff
@@ -35,7 +35,7 @@ type Config struct {
 }
 
 // New creates an instance of the KMS Service Server.
-func New(ctx context.Context, pc *Config) (*KeyManagementServiceServer, error) {
+func New(_ context.Context, pc *Config) (*KeyManagementServiceServer, error) {
```

Comment:
If context is not used here, let's remove it!

Reply:
I intend to figure out the use of context in https://github.com/Azure/kubernetes-kms/pull/210 hence did not remove it here.

Reply:
Adding it back in #210 if it's used would be fine but given it's not used now, lets remove it.